pf/tfbridge/provider.go

```diff
@@ -148,7 +148,7 @@ func newProviderWithContext(ctx context.Context, info ProviderInfo,
 	}, nil
 }
 
-func newProviderServer(
+func NewProviderServer(
```

Comment:
Should we mark this as an unstable API. This isn't really intended for end users to consume AFAIK.